MCDataProducts/inc/KalSeedMC.hh

```diff
@@ -34,15 +34,17 @@ namespace mu2e {
     MCRelationship _rel; // relationship of this particle to its primary
     uint16_t _nhits; // number of associated StrawHits
     uint16_t _nactive; // number of associated active hits
-    XYZVectorF _mom; // initial momentum
+    CLHEP::HepLorentzVector   _mom; // initial momentum
```

Comment:
These 3 items are accessible via the SimParticle?  Why do you need  copies here?

Reply:
its for TrkAna use. I dont believe we have direct access to these (as with the other things which are explicitly added here)

Reply:
Can you explain a little more.  Is that issue that are not keeping the right data products when we write out events for TrkAna?  Or did we compress the information away?  Without knowing more it seems that a better answer is to make sure that we write out everything needed for TrkAna instead of making a second copy?

Reply:
I added some code to TrkAna to add in the true track parameters and it uses this data product

Reply:
What I don't understand is why you need a copy.  The information is available by following the Ptr.

Reply:
This question is beyond the scope of this PR, but...
There is no stored ptr, SimPartStub saves only the key.  It can rebuild the Ptr if you have the collection (handle).  
The design intent is to be a low-payload summary of a SimParticle, hence storing the momentum.